pkg/kando/location.go

```diff
@@ -20,14 +20,26 @@ import (
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 
+	"github.com/kanisterio/kanister/pkg/datamover"
 	"github.com/kanisterio/kanister/pkg/param"
 )
 
 const (
-	pathFlagName    = "path"
-	profileFlagName = "profile"
+	pathFlagName                                = "path"
+	profileFlagName                             = "profile"
+	repositoryServerFlagName                    = "repository-server"
```

Comment:
```suggestion
	repositoryServerFlagName                    = "repository-server"

```
Add new line so that gofmt aligns to columns correctly

Reply:
Done ! Added comments as well for `DataMoverTypeProfile` and `DataMoverTypeRepositoryServer`